lib/wallet/src/Cardano/Wallet/DB/Sqlite/Migration.hs

```diff
@@ -111,7 +116,7 @@ data InvalidDatabaseSchemaVersion
     deriving (Show, Eq, Exception)
 
 currentSchemaVersion :: SchemaVersion
-currentSchemaVersion = SchemaVersion 2
+currentSchemaVersion = SchemaVersion 3
```

Comment:
```suggestion
currentSchemaVersion = SchemaVersion 2
```

As the schema itself did not change, only our bugs associated with it. 🤷 Up to you.

Reply:
Bugs have no right to change the schema. One day this will change! 
-- Bugs liberation front

Reply:
🐞🐞🐞